Resources/Prototypes/Entities/Markers/drop_pod_markers.yml

```diff
@@ -0,0 +1,71 @@
+- type: entity
+  name: DropPod Beacon
```

Comment:
Don't add tokens that players can't interact with. We have Beacons that store the name of locations and are mapped to all stations, use them.